sno/gpkg.py

```diff
@@ -234,6 +241,76 @@ def geom_to_ogr(gpkg_geom, parse_srs=False):
     return geom
 
 
+def ogr_to_geom(ogr_geom):
```

Comment:
also need roundtrip tests for this and `geom_to_ogr` I think; I'll add some

Reply:
done in 3811cdc5538ea463f61521c0cc9227c71e76f368

Reply:
>  need to compare this against the matching code in the postgis branch.

tried to look at this, not quite sure what you mean, could you link? I grepped the postgis source code for 'geopackage' but didn't find any geometry converters there.

Reply:
ohhh you're referring to the `postgis1-working-copy1` branch of sno itself 👍 

Didn't realise there was similar code there; will look at it

Reply:
that's... duplicate code. Wish I'd known that was there earlier

Reply:
I've made some changes in a8d8fc364947f779d29aec3f287c892c9ca6c065 based on that (removed envelopes; made one `struct.pack()` call instead of multiple) We can de-dupe the implementations once this is merged.